.ci/Jenkinsfile

```diff
@@ -12,10 +12,28 @@
 import static com.r3.build.BuildControl.killAllExistingBuildsForJob
 killAllExistingBuildsForJob(env.JOB_NAME, env.BUILD_NUMBER.toInteger())
 
+def isReleaseTag() {
+    return (env.TAG_NAME =~ /^release-.*$/)
+}
+
+def isReleaseCandidate() {
+    return (isReleaseTag()) && (env.TAG_NAME =~ /.*-(RC|HC)\d+(-.*)?/)
+}
+
+def isReleaseBranch() {
+    return (env.BRANCH_NAME =~ /^release\/.*$/)
+}
+
+def isRelease = isReleaseTag() || isReleaseCandidate()
+String publishOptions = (isReleaseBranch() || isRelease) ? "-s --info" : "--no-daemon -s -PversionFromGit"
```

Comment:
`versionFromGit` i supect is not needed here, unless the gradle code relies on it which I can not see from a quick look